src/analytics/analytics.js

```diff
@@ -5,6 +5,19 @@ import LoggingService from '../services/LoggingService';
 
 const eventLogApiBaseUrl = `${configuration.LMS_BASE_URL}/event`;
 
+
+// Uses a JSON as a source of key value pairs for x-www-form-urlencoded output.
+// Note should be handled by axios, but is not built in.
+// - see https://github.com/axios/axios/issues/97
```

Comment:
@douglashall: Not sure if we can move this into frontend-auth apiClient setup?  I'll discuss more.

Reply:
I'd probably go with something like this instead: https://www.npmjs.com/package/form-urlencoded

Reply:
@douglashall: I’ll discuss tomorrow the possible use of this library for a more general solution with axios. Merged this as-is to fix big and can iterate. Thanks.